packages/http-server/src/HttpServerModule.ts

```diff
@@ -113,9 +114,21 @@ export abstract class HttpServerModule<
 					fullPath = fullPath.slice(0, -1);
 				}
 
+				const parametersConfig = await this.createParametersConfigurations({
+					controllerReflection,
+					methodReflection
+				});
+
+				this.routes.push({
```

Comment:
collecting the list of endpoints into the routes array, that acts as source of truth